Contents/Code/siteRealityLovers.py

```diff
@@ -0,0 +1,121 @@
+import PAsearchSites
+import json
+import re
+
+def search(results, encodedTitle, title, searchTitle, siteNum, lang, searchByDateActor, searchDate, searchSiteID):
+    Log('******SEARCH CALLED*******')
+    if searchSiteID != 9999: siteNum = searchSiteID
+
+    url = PAsearchSites.getSearchSearchURL(siteNum)
+    post_values = {
+        "sortBy": "MOST_RELEVANT",
+        "searchQuery": searchTitle,
+        "categoryId": "null",
+        "perspective": "null",
+        "actorId": "null",
+        "offset": "0",
+        "isInitialLoad": "true",
+        "videoView": "MEDIUM",
+        "device": "DESKTOP"
+    }
```

Comment:
Can be shorted to
```json
{
    "sortBy": "MOST_RELEVANT",
    "searchQuery": searchTitle,
    "videoView": "MEDIUM"
}
```
`sortBy` not necessary too, but it can be useful